pkg/apis/core/types.go

```diff
@@ -2741,6 +2741,31 @@ type ContainerStatus struct {
 	// +optional
 	// +featureGate=RecursiveReadOnlyMounts
 	VolumeMounts []VolumeMountStatus
+	// User represents user identitiy information of the first process in the container
```

Comment:
typo: identity

Reply:
addressed in [5f0d761](https://github.com/kubernetes/kubernetes/pull/117842/commits/5f0d7613520d87745a229f8cc854bbf0e2e18221)